### PR TITLE
Japan's Shinto fix

### PR DESCRIPTION
**Pull request by EliteNinjaX**
Japan and her puppets are now Shinto, and some pops have changed to created better balance, lore and colour for East Asia.
Please disregard and delete the other pull request for Japan and Venice as I was still learning how they worked.

### CODE REVIEW

**Review by MadJamez — CHANGES_REQUESTED** [review]
Remove color changes for now. Mughals green like in FF.
Best to make a different branch for the other color changes.

**Comment by Xerodoeht** [comment]
The pop changes are definitely pretty questionable at points as well, I noticed you added, a lot of Chinese Catholics? I think you should definitely wait on doing that until there is a justification for them being there

File: localization/english/dvg_countries_l_english.yml [hunk 1]
```diff
@@ -393,4 +393,13 @@
...
- dyn_c_great_jing_heavenly_kingdom: "Great Jing Heavenly Kingdom"
\ No newline at end of file
+ KUR: "Kurdistan"
+ KUR_ADJ: "Kurdistanian"
+ SHU:0 "China"
+ SHU_ADJ:0 "Chinese"
+ dyn_c_great_shun:0 "Great Shun"
+ dyn_c_great_shun_adj:0 "Shun"
+ JIA:0 "Jiangxi"
+ JIA_ADJ:0 "Jiangxi" #No idea tbh
```
Comment (Xerodoeht) [inline localization/english/dvg_countries_l_english.yml:403]: Could use gan?

File: localization/english/dvg_flavor_china_l_english.yml [hunk 2]
```diff
@@ -2,6 +2,22 @@
  #Journal Entries
  je_china_reunification:0 "China Reunification"
  je_china_reunification_reason:0 "China Reunification"
+ decline_of_great_shun_main:0 "The Decline of Great Shun"
+ 
+ #Laws
+ law_china_feudal_armies:0 "Weisuo System"
+ law_china_feudal_armies_desc:0 "Originally developed by the Yuan, the Ming and latter Shun inherited the Wei-Suo system to help guard it's borders, by creating military households along the frontiers. However it is horribly out of date as soldiers are more loyal to thir officers-latrons rather than the throne and as such uninterested in warfare, preventing any type of efficient mobilisation."
```
Comment (Xerodoeht) [inline localization/english/dvg_flavor_china_l_english.yml:9]: Is officer-latron correct here?

File: localization/english/dvg_countries_l_english.yml (hunk 1 above)
Comment (Xerodoeht) [reply localization/english/dvg_countries_l_english.yml:403]: Though Jiangxi is fine by the way, you don't need to change it

**Review by MadJamez — CHANGES_REQUESTED** [review]
Not sure if the review description is correct. Can you maybe update it, and if this includes multiple features, please split it up over multiple reviews.

**Pull request closed without merging**